src/app/style.rs

```diff
@@ -1,8 +1,11 @@
 use clap::ValueEnum;
+use core::panic;
```

Comment:
No need to import stuff from `core`.

Reply:
Ignore this, I was using panic for dumb testing.  How can I enable tracing logging to STDOUT btw?

Reply:
`RUST_LOG=trace cargo r` doesn't help as the TUI takes over STDOUT

Reply:
There is a logging widget in the TUI: https://github.com/orhun/gpg-tui?tab=readme-ov-file#show-logs

You can also save logs to a file.

Reply:
Thanks!! That's just neat